.pre-commit-config.yaml

```
@@ -11,14 +11,14 @@ repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
     rev: v5.0.0
     hooks:
-      - id: trailing-whitespace
-      - id: end-of-file-fixer
-      - id: mixed-line-ending
-        args:
-          - --fix=lf
+      - id: check-added-large-files
       - id: check-toml
       - id: check-yaml
-      - id: check-added-large-files
+        args: [ --allow-multiple-documents, --unsafe ]
+      - id: end-of-file-fixer
+      - id: mixed-line-ending
+        args: [ --fix=lf ]
+      - id: trailing-whitespace
```

There are a few more that might be usefull imo.
What do you think?

```suggestion
      - id: trailing-whitespace
      - id: check-executables-have-shebangs
      - id: check-shebang-scripts-are-executable
        exclude: '.+\.rs' # would be triggered by #![doc = include_str!("../README.md")]
      - id: check-symlinks
      - id: destroyed-symlinks
```